client/components/Book.jsx

```diff
@@ -32,54 +40,112 @@ class Book extends React.Component {
 
   handleChangeDate (evt, date) {
     this.setState({
-      date: moment(date).format('DD/MM/YYYY')
+      date: moment(date).format('YYYY/MM/DD')
     })
   }
 
-  handleChangeEvent (date, key) {
+  handleChangeTime (evt, key) {
+    console.log(evt.target.value)
     this.setState({
-      [key]: moment(date).format('h:mmA')
+      [key]: evt.target.value
     })
   }
 
   handleSubmit (evt) {
     evt.preventDefault()
-    // this.props.dispatch(saveNewCaption(this.state, (newCaptionId) => {
-    //   this.props.routerProps.history.push(`/images/${this.state.imageId}/${newCaptionId}`)
-    // }))
-    console.log(this.state)
+    this.setState({
+      fullName: this.props.fullName,
+      email: this.props.emailAddress,
+      phoneNumber: this.props.phoneNumber,
+      authId: this.props.authId
+    }, () => {
```

Comment:
What happens if the user changes the name, email or phone number text fields?  The props values would still be saved into the state.  Consider something like componentWillReceiveProps() https://facebook.github.io/react/docs/react-component.html#componentwillreceiveprops